framework/test/mocha/functional/http/get/transactions.js

```diff
@@ -356,6 +357,120 @@ describe('GET /api/transactions', () => {
 						});
 					});
 			});
+
+			describe('asset field', () => {
+				it('using type 0 should return asset field with correct properties', async () => {
```

Comment:
I don't think we need async statement since we already returning promise right away. The same usage reoccurs in the tests below.

Reply:
Also, I'd prefer to keep async and instead of doing assertions in the then callback, I'd simply await the request.